composer.json

```diff
@@ -22,12 +22,12 @@
 	},
 	"require" : {
 		"php" : ">=8.1",
-		"typo3/cms-core" : "^12",
-		"typo3/cms-rte-ckeditor" : "^12",
-		"typo3/cms-fluid-styled-content" : "^12",
-		"typo3/cms-beuser" : "^12",
-		"typo3/cms-backend" : "^12",
-		"typo3/cms-scheduler" : "^12",
+		"typo3/cms-core" : "^12 || ^13",
+		"typo3/cms-rte-ckeditor" : "^12 || ^13",
+		"typo3/cms-fluid-styled-content" : "^12 || ^13",
+		"typo3/cms-beuser" : "^12 || ^13",
+		"typo3/cms-backend" : "^12 || ^13",
+		"typo3/cms-scheduler" : "^12 || ^13",
```

Comment:
Hier kannst du 12 rausnehmen, ich glaube 12 sollte nicht mehr funktionieren mit den Änderungen oder?

Reply:
Ich teste es grad mal

Reply:
![image](https://github.com/user-attachments/assets/7f1230b0-a533-46b5-b4af-fa2314448b46)
Scheint nicht zu gehen, wird rausgenommen

Reply:
Fixed in 5b155f367e45496867d9aa06a8f89318824759c6